README.md

```diff
@@ -1 +1,12 @@
-# opensuse-jeptalon-base
\ No newline at end of file
+# openSUSE and JepTalon image
+
+This project builds on the openSUSE Python2 image [here](https://github.com/CAFapi/opensuse-python2-image), Talon and Jep are then installed on the image. This image can be used as a base by projects which require Python2, Talon and Jep.
+
```

Comment:
Can you add a brief description of what Jep and Talon are as well as links to them?